### PR TITLE
add svgpp library

### PR DESCRIPTION
**Pull request by maksim-petukhov**
Specify library name and version:  **svgpp/cci.20221030**

https://github.com/svgpp/svgpp

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by valgur — APPROVED** [review]

**Review by czoido — APPROVED** [review]

File: recipes/svgpp/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return 11
+
+    def layout(self):
+        basic_layout(self)
```
Comment (czoido) [inline recipes/svgpp/all/conanfile.py:28]:
```suggestion
        basic_layout(self, src_folder="src")
```

**Review by czoido — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Comment by maksim-petukhov** [comment]
`continuous-integration/jenkins/pr-merge — This commit cannot be built` - why?

**Comment by czoido** [comment]
> `continuous-integration/jenkins/pr-merge — This commit cannot be built` - why?

Sorry @maksim-petukhov , we are doing some upgrades to the infra and all the system it's a bit unstable. We'll try to be back to normal soon.

**Comment by maksim-petukhov** [comment]
Thank you for your answer @czoido. I think you (I mean the conan team) really need to communicate with the community. There is a conan Slack group with a lot of members and it would be great to mention those upgrades there beforehand.

**Comment by czoido** [comment]
> Thank you for your answer @czoido. I think you (I mean the conan team) really need to communicate with the community. There is a conan Slack group with a lot of members and it would be great to mention those upgrades there beforehand.

Sorry for that, it was a routine operation that caused unexpected problems on the Jenkins side, a series of errors concatenated that affected all the open PR's. We will try to communicate better in the future, thanks a lot for your patience.

**Pull request merged**